content/en/docs/concepts/storage/volumes.md

```diff
@@ -190,6 +190,11 @@ To disable the `azureFile` storage plugin from being loaded by the controller ma
 and the kubelet, set the `InTreePluginAzureFileUnregister` flag to `true`.
 
 ### cephfs
+{{< feature-state for_k8s_version="v1.28" state="deprecated" >}}
+
+{{< note >}}
+We recommend to use CephFS CSI out-of-tree driver instead.
```

Comment:
We [avoid using “we”](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-using-we). Please reword, eg “The Kubernetes project recommends&hellip;”